### PR TITLE
SideBarButton set text below the icon

### PR DESCRIPTION
**Pull request by BaraMGB**
this would fix https://github.com/LMMS/lmms/issues/3234#issuecomment-271785914
2017-01-13-104015_1920x1080_scrot

### CODE REVIEW

**Comment by mikobuntu** [comment]
:+1:  great job :)

**Comment by tresf** [comment]
Unrelated to this PR, but the side-tabs needs more contrast from each other.  The description is indistinguishable from the two icons.

**Comment by mikobuntu** [comment]
@tresf something like this design in my mockup could work?
sidebar

**Comment by tresf** [comment]
> @tresf something like this design in my mockup could work?

👍

**Comment by BaraMGB** [comment]
> something like this design in my mockup could work?

I guess this one can easily made in the styles.css

**Comment by mikobuntu** [comment]
@BaraMGB Yes i was thinking the same. I could probably dig out my css book and have a look to see how to do this, well along with qtcreator :) , tho I hope the experts step in and do this i.e @RebeccaDeField et all :)

**Comment by BaraMGB** [comment]
in the styles.css line 497 changed to:
`	background: #2f353b;`
looks like:
2017-01-13-201705_1280x800_scrot

Should I add this to this PR?

**Comment by mikobuntu** [comment]
@BaraMGB did adding the background conflict with the original PR, as i see the icon is now below the text in your last image?

**Comment by BaraMGB** [comment]
I changed the styles.css in master for be quick. ;)

**Comment by BaraMGB** [comment]
2017-01-13-214328_1280x800_scrot

looks not bad.

**Comment by tresf** [comment]
Better but still requires more contrast.  The Instrument Buttons are about as close as we should allow (despite the similarities in color, they have additional bevel and shading which makes them standout from the background).


image

**Comment by RebeccaDeField** [comment]
This is looking good, people :) I do agree with @tresf that if we are adding this to enhance the contrast, we might want to step up the contrast a bit more.

**Comment by RebeccaDeField** [comment]
Here are some colors that should work well for the sidebar tab.

**Horizontal gradient:**
Stop 1: #4b525b 
Stop 2: #2f333a

Should look something like this


@BaraMGB

**Comment by BaraMGB** [comment]
Thanks to @RebeccaDeField :+1:  

2017-01-23-131900_1920x1080_scrot

File: data/themes/default/style.css [hunk 1]
```diff
@@ -494,7 +494,7 @@ SideBar QToolButton:pressed {
 }
 
 SideBar QToolButton:checked {
-	background: none;
+        background: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 #4b525b,  stop:1 #2f333a);
```
Comment (tresf) [inline data/themes/default/style.css]: @BaraMGB code style formatting... You've used spaces instead of a tab.

File: src/gui/widgets/SideBar.cpp [hunk 2]
```diff
@@ -110,8 +110,9 @@ SideBar::~SideBar()
 void SideBar::appendTab( SideBarWidget *widget )
 {
 	SideBarButton *button = new SideBarButton( orientation(), this );
-	button->setText( widget->title() );
+	button->setText( " " + widget->title() );
```
Comment (tresf) [inline src/gui/widgets/SideBar.cpp]: Can you explain this?  Is this for deference purposes?

Comment (tresf) [inline src/gui/widgets/SideBar.cpp]: Never mind, it's a space, I thought it was empty.  Please disregard.

**Comment by tresf** [comment]
Just one minor code formatting change, then we can merge this.

**Comment by BaraMGB** [comment]
done ;)

**Pull request merged**